main.cpp

```diff
@@ -52,106 +52,144 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: athlete
+
+//  action 1: the athlete runs
+athlete.run();
+
+//  action 2: the athlete jumps
+athlete.jump();
+    
+//  action 3: the athlete rests
+athlete.rest();
+
+    
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: eye
+    
+//  action 1: eyes blink
+eye.blink();
 
-//  action 2:
+//  action 2: eyes produce tears
+    
+eye.cry();
 
-//  action 3:
+//  action 3: eyes stare into something
+
+eye.stare();
+    
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Calculator
+    
+//  action 1: Calculator adds numbers
+calculator.add();
+    
+//  action 2: Calculator enters into radian mode
+calculator.switchToRadian();
+
+//  action 3: Calculator powers off
+calculator.shutDown();
+
+//  4) 
+//  Noun: programmer 
+    
+//  action 1: the programmer writes code
+programmer.writeCode();
+    
+//  action 2: the programmer debugs the code
+programmer.debug();
+    
+//  action 3: the programmer reviews the code
+programmer.review();
 
 //  5)
-//  Noun:
-//  action 1:
+//  Noun: ipod
+//  action 1: ipod plays songs
+ipod.playMusic();
 
-//  action 2:
+//  action 2: ipod shuffles the playlist
+ipod.shufflePlaylist();
 
-//  action 3:
+//  action 3: ipod changes volume
+ipod.adjustVolume();
 
 //  6)
-//  Noun:
-//  action 1:
+//  Noun: teeth
+//  action 1: using teeth to bite
+teeth.bite();
 
-//  action 2:
+//  action 2: using teeth to smile
+teeth.smile();
 
-//  action 3:
+//  action 3: using teeth to chew things
+teeth.chew();
 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun: grandmother
+//  action 1: grandmother cooks
+grandmother.cook();
 
-//  action 2:
+//  action 2: grandmother prays
+grandmother.pray();
 
-//  action 3:
+//  action 3: grandmother watches tv
+grandmother.watchTv();
 
 //  8)
-//  Noun:
-//  action 1:
+//  Noun: lion
+    
+//  action 1: the lion hunts
+lion.hunt();
 
-//  action 2:
+//  action 2: the lion sleeps
+lion.sleep();
 
-//  action 3:
+//  action 3: the lion roars
+lion.roar();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.